edge/pkg/edged/edged_pods.go

```diff
@@ -708,7 +708,13 @@ func (e *edged) podFieldSelectorRuntimeValue(fs *v1.ObjectFieldSelector, pod *v1
 		return pod.Spec.NodeName, nil
 	case "spec.serviceAccountName":
 		return pod.Spec.ServiceAccountName, nil
-	// TODO: Add status.hostIP here
+	case "status.hostIP":
+		hostIP, err := e.getHostIPByInterface()
+		if err != nil {
+			klog.Errorf("Failed to get host IP: %v", err)
```

Comment:
I suggest deleting `klog.Errorf("Failed to get host IP: %v", err)`, otherwise err will be logged twice

Reply:
Thanks for your suggestion. I will resubmit the pull request after modification.